src/features/MentorPage/components/MentorList/MentorCard/Expanded/Header.tsx

```diff
@@ -1,21 +1,25 @@
-import styled from 'styled-components';
+import type { Mentor } from '@/features/MentorPage/mentorPageApi';
+
+import { useMobileMode } from '@/hooks/useMobileMode';
+
 import ProfilePicPlaceholder from '@/static/icons/chat-profilepic.svg';
 import { breakpoints, palette } from '@/components/variables';
+import { IconButton } from '@/components/Buttons';
 import { BasicInfo } from './BasicInfo';
-import { Mentor } from '@/features/MentorPage/mentorPageApi';
-import { useMobileMode } from '@/hooks/useMobileMode';
 import Text from '@/components/Text';
+import styled, { css } from 'styled-components';
 
 type Props = {
   mentor: Mentor;
+  onDismiss: () => void;
 };
 
-export const Header = ({ mentor }: Props) => {
+export const Header = ({ mentor, onDismiss }: Props) => {
   const availabilityMessage = mentor.isVacationing ? 'Ei tavoitettavissa' : '';
   const isMobile = useMobileMode();
 
   return isMobile ? (
-    <Container isLoggedIn={!mentor.isVacationing}>
+    <Container isLoggedIn={!mentor.isVacationing} isMobile>
```

Comment:
I had to think about this for a sec... How is the container logged in? 😄  I think this attribute could be renamed to something more fitting.

Could it be as simple as 

`<Container mentorIsVacationing={mentor.isVacationing} isMobile>
`
and then in the styled Container:

`mentorIsVacationing ? palette.blueGray : palette.purple
`
What do you think?

Reply:
Ah of course, `isAvailable` is used elsewhere to indicate this! So maybe we could use it instead!

Reply:
Yeah, i think the isLoggedIn was something that was added in the mock-stage and somehow I overlooked it :p